.github/workflows/ci-java.yml

```diff
@@ -1,11 +1,38 @@
 name: 'CI - Java'
 on: pull_request
 jobs:
-  mvn-job:
+  mvn-fmt-maven-plugin:
+    runs-on: macos-latest
```

Comment:
Why do we use a MacOS runner for the CI job? I see it was that way previously, but would an Ubuntu virtual environment make more sense to keep it closer to how things will run in Jenkins and deployed envs?

Reply:
I believe that we switched to that a while back, to either workaround or reliably reproduce (I don't recall) a platform-specific bug.